client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -364,8 +370,22 @@ function getRedirectUrlForConciergeNudge( {
 				}
 			}
 
+			if ( hasBusinessPlan( cart ) ) {
+				if ( shouldShowDifmUpsell ) {
+					recordTracksEvent( 'calypso_eligible_difm_upsell' );
```

Comment:
Same as above. Custom Tracks event is created to be used as an exposure event for the experiment. This helps give more control on which users we want to enter into the experiment.